src/components/button/_button.scss

```diff
@@ -230,20 +230,4 @@
       background-size: 30px 19px;
     }
   }
-
```

Comment:
I think we'd likely tie the old font to legacy mode, so we'd keep the old stuff around but only when compatibility flags are set.